README.md

```diff
@@ -40,14 +40,110 @@ See [LevelDB Persistence](https://github.com/yjs/y-leveldb) for more info.
 PORT=1234 YPERSISTENCE=./dbDir node ./node_modules/y-websocket/bin/server.js
 ```
 
-### Scaling
+#### Start a secure websocket server
+
+You can also use the secure `wss://` protocol, which like `https://`, uses SSL/TLS.  For this, you first need a certificate.  If your server is  using `https://`, you will already have a certificate.  If not, you can obtain one for free from [certbot](https://certbot.eff.org/).  You should then provide the directory in which the certifcate and private key are to be found:
+
+```sh
+PORT=1234 YPERSISTENCE=./dbDir CERTDIR=/etc/letsencrypt/live/example.com node ./node_modules/y-websocket/bin/server.js
```

Comment:
I remember that one of my first contributions to OSS was support for secure websockets to a signaling server.  

Now I honestly don't want to recommend users to maintain certificates manually anymore. This comes with a lot of security concerns. It is definitely possible, and might even reduce some overhead. But nodejs is not a secure environment to handle certificates. I know that a lot of novice users use eval to evaluate responses from clients (e.g. if you use jsdom). It is entirely possible that this approach will leak certificates to clients.

As a library author, I have the responsibility to recommend secure approaches. I don't have a lot of confidence in nodejs handling certificates. I know that it can be secure, and that it is important that nodejs supports wss. Nevertheless, I can't generally recommend this approach to new users because I think that this should be handled by a reverse proxy like nginx or apache.